src/test/java/quentin/core/ChainContainerShould.java

```diff
@@ -2,13 +2,13 @@
 
 import org.junit.jupiter.api.Test;
 
-import java.util.List;
+import java.util.Set;
 import java.util.stream.Stream;
 
 import static org.junit.jupiter.api.Assertions.*;
 import static quentin.core.Position.in;
 
-public class ChainShould {
+public class ChainContainerShould {
 
     @Test
     public void touchEdgesIfItIsACompleteChain() {
```

Comment:
ChainContainerShould.touch`Edges`IfItIsACompleteChain should be touch`Sides`IfItIsACompleteChain.